app/src/pages/OnDeviceDisplay/ProtocolDashboard/PinnedProtocol.tsx

```diff
@@ -18,55 +22,114 @@ import { LongPressModal } from './LongPressModal'
+  const displayedName = truncateString(
+    protocolName,
+    cardSize === 'full' ? 115 : 90,
+    cardSize === 'full' ? 82 : 45
+  )
+  const { t } = useTranslation('protocol_info')
 
-  const longpress = useLongPress()
+  const cardStyleBySize: {
+    [s in CardSizeType]: {
+      fontSize: string
+      height: string
+      lineHeight: string
+      width: string
+    }
+  } = {
+    full: {
+      fontSize: TYPOGRAPHY.fontSize32,
+      height: '100%',
+      lineHeight: TYPOGRAPHY.lineHeight42,
+      width: '100%',
+    },
+    half: {
+      fontSize: TYPOGRAPHY.fontSize28,
+      height: '10.75rem',
+      lineHeight: TYPOGRAPHY.lineHeight36,
+      width: '29.25rem',
+    },
+    regular: {
+      fontSize: TYPOGRAPHY.fontSize28,
+      height: '10.75rem',
+      lineHeight: TYPOGRAPHY.lineHeight36,
+      width: '28.375rem',
+    },
+  }
 
   const handleProtocolClick = (
     longpress: UseLongPressResult,
     protocolId: string
   ): void => {
-    if (!longpress.isLongPressed) {
+    if (longpress.isLongPressed !== true) {
       history.push(`/protocols/${protocolId}`)
     }
   }
 
   return (
     <Flex
-      gridGap={SPACING.spacing3}
-      height="13rem"
       alignItems={ALIGN_FLEX_START}
-      padding={SPACING.spacing4}
-      onClick={() => handleProtocolClick(longpress, protocol.id)}
-      ref={longpress.ref}
-      border={`1px ${COLORS.medGreyHover} solid`}
-      maxWidth="18.125rem"
-      minWidth="18.125rem"
+      backgroundColor={COLORS.light_one}
+      borderRadius={BORDERS.size_four}
       flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing3}
+      height={cardStyleBySize[cardSize].height}
       justifyContent={JUSTIFY_SPACE_BETWEEN}
+      maxWidth={cardStyleBySize[cardSize].width}
+      minWidth={cardStyleBySize[cardSize].width}
+      onClick={() => handleProtocolClick(longpress, protocol.id)}
+      padding={SPACING.spacing4}
+      ref={longpress.ref}
     >
       <StyledText
-        key={protocol.id}
-        fontSize="1.5rem"
-        lineHeight="2.0625rem"
+        fontSize={cardStyleBySize[cardSize].fontSize}
         fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+        lineHeight={cardStyleBySize[cardSize].lineHeight}
       >
-        {protocolName}
+        {displayedName}
       </StyledText>
-      <StyledText
-        fontSize="1rem"
-        lineHeight="1.125rem"
-        fontWeight={TYPOGRAPHY.fontWeightRegular}
+      <Flex
+        flexDirection={DIRECTION_ROW}
+        gridGap={SPACING.spacing3}
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
+        width="100%"
       >
-        {format(new Date(protocol.createdAt), 'MMM Io, p')}
-      </StyledText>
-      {longpress.isLongPressed && (
+        <StyledText
+          fontSize={TYPOGRAPHY.fontSize22}
+          fontWeight={TYPOGRAPHY.fontWeightRegular}
+          lineHeight={TYPOGRAPHY.lineHeight28}
+        >
+          {lastRun !== undefined
+            ? `${String(t('last_run'))} ${formatDistance(
```

Comment:
I guess you added `String()` to avoid warning.
If so, we can remove `String()`.  I talked about this with Brian a couple of weeks ago and his suggestion was to handle this warning by updating eslint config. (Unfortunately, we haven't updated that for a while...) 

```suggestion
            ? `${t('last_run'))} ${formatDistance(
```

Reply:
Yeah... eslint is not happy if you try to pass in a number as a string. I can take these out, but there are 585 total uses of this pattern in 135 files in the repo.